build.sbt

```diff
@@ -24,23 +25,23 @@ releaseProcess := Seq[ReleaseStep](
   checkSnapshotDependencies,
   inquireVersions,
   runClean,
-  runTest,
```

Comment:
As `gha-scala-library-release-workflow` can't provide the AWS credentials required by the tests - we've got to forego the tests during release.

* https://github.com/guardian/gha-scala-library-release-workflow/issues/42